### PR TITLE
Expose services, and use latest revisions

### PR DESCRIPTION
**Pull request by lazypower**
Fixes for #5 and #6

### CODE REVIEW

**Comment by lazypower** [comment]
@mbruzek need a review when you get time on this one

File: bundle.yaml [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
-    charm: "cs:~containers/trusty/topbeat-8"
+    charm: "cs:trusty/topbeat-1"
     annotations:
       "gui-x": "819.045"
       "gui-y": "585.538"
   kibana:
-    charm: "cs:~containers/trusty/kibana-1"
+    charm: "cs:trusty/kibana-12"
     num_units: 1
+    expose: true
```
Comment (arosales) [inline bundle.yaml:16]: Per #6 needs:

```
    options:
      dashboards: "beats"
```

**Comment by arosales** [comment]
Per #6 the readme also needs to include instructions on how to access the dashboard. Similar to
https://api.jujucharms.com/charmstore/v5/bundle/observable-kubernetes-0/archive/README.md

## Access Kibana dashboard

The Kibana dashboard can display real time graphs and charts on the details of
the cluster. The Beats charms are sending metrics to the Elasticsearch and
Kibana displays the data with graphs and charts.

Get the charm's public address from the `juju status` command.
- Access Kibana by browser:  http://KIBANA_IP_ADDRESS/
- Select the index pattern that you want as default from the left menu.
  - Click the green star button to make this index a default.
- Select "Dashboard" from the Kibana header.
  - Click the open folder icon to Load a Saved Dashboard.
- Select the "Topbeat Dashboard" from the left menu.

Setup Kibana

File: bundle.yaml (hunk 1 above)
Comment (lazypower) [inline bundle.yaml:16]: Looks like i missed a push. I have removed the artifact and added that option. 1 sec and i'll get this addressed

**Comment by AdamIsrael** [comment]
⛵

**Pull request merged**